app/repositories/sift/ApplicationSiftRepository.scala

```diff
@@ -301,5 +303,53 @@ class ApplicationSiftMongoRepository(
       else { () }
     }
   }
+
+  def fixDataByRemovingSiftEvaluation(applicationId: String): Future[Unit] = {
+    val query = BSONDocument("applicationId" -> applicationId)
+
+    val updateOp = bsonCollection.updateModifier(
+      BSONDocument(
+        "$set" -> BSONDocument("applicationStatus" -> ApplicationStatus.SIFT),
+        "$unset" -> BSONDocument(s"testGroups.$phaseName" -> "")
```

Comment:
@henricook this un-sets the entire sift phase. I think we want to remove the result for a particular scheme here.

Reply:
The ticket hasn't specified a particular scheme, and i believe we talked about removing their whole evaluation so they could just be re-sifted

Reply:
The ticket contains two candidates, one who needs an individual scheme red'ing and one who needs to be rolled back from AC_AWAITING_ALLOCATION

Reply:
The currentSchemeStatus also needs resetting, i've had some crossover here so took care of it in the PR for FSET-1909